### PR TITLE
@thunderstore/dapper:  add three methods to replace getTeam

### PR DESCRIPTION
**Pull request by anttimaki**

### CODE REVIEW

**Comment by anttimaki** [comment]
Current dependencies on/for this PR:
* master
  * **PR #857** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/857" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/857?utm_source=stack-comment).

**Comment by anttimaki** [comment]
CI pipeline will fail until https://github.com/thunderstore-io/Thunderstore/pull/892 is deployed to test environment.

File: packages/thunderstore-api/src/fetch/teamDetails.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { RequestConfig } from "../index";
+import { apiFetch } from "../apiFetch";
+
+export async function fetchTeamDetails(
+  config: RequestConfig,
+  teamName: string
+) {
+  const path = `api/cyberstorm/team/${teamName}/`;
```
Comment (anttimaki) [inline packages/thunderstore-api/src/fetch/teamDetails.ts:8]: Would it improve caching if we'd use `${teamName.toLowerCase()}` here and in other similar places? I'm not going to implement it here though, it should be done in one place when we get around implementing the "route definitions" helper.

Comment (MythicManiac) [reply packages/thunderstore-api/src/fetch/teamDetails.ts:8]: Assuming we do caching on the load balancing layer for this view, yes it would

**Review by Oksamies — APPROVED** [review]

File: packages/thunderstore-api/src/fetch/teamServiceAccounts.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { RequestConfig } from "../index";
+import { apiFetch } from "../apiFetch";
+
+export async function fetchTeamServiceAccounts(
+  config: RequestConfig,
+  teamName: string
+) {
+  const path = `api/cyberstorm/team/${teamName}/service_accounts/`;
```
Comment (MythicManiac) [inline packages/thunderstore-api/src/fetch/teamServiceAccounts.ts:8]: `service_accounts` -> `service-accounts`

**Review by MythicManiac — APPROVED** [review]
Needs a single change, looks good to me beyond that

**Pull request merged**